fudge-ratio/src/main/webapp/about/about.css

```diff
@@ -2,52 +2,203 @@
 @import url('https://fonts.googleapis.com/css2?family=Bad+Script&family=Open+Sans:wght@400;700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&family=Poiret+One&display=swap');
 :root {
-  --background-color: rgb(224, 239, 241);
-  --font-color: rgb(30, 30, 30);
+  --background-color: rgb(229, 231, 225);
   --accent-color:rgba(45, 93, 122, 0.712);
+  --block-color:rgb(158, 158, 2);
+  --base: min(3.5vw, 1.7rem);
 }
-
 body { 
-  margin:0; 
+  margin:0;
   background: var(--background-color);
 }
-
+/* Style for the bar on top and text on top*/
 .banner {
-  overflow: hidden;
-  /* 161, 111, 32, 0.712*/
+  z-index: 3;
+  word-wrap: break-word;
+  backdrop-filter: blur(2rem);
   background-color: rgba(45, 93, 122, 0.712);
   font-family: 'Poiret One', cursive;
   position: fixed;
   top: 0;
-  width: 100vw;
-  height: 15vh;
+  width: 100%;
+  height: 12vh;
+  padding-top:2%;  
+  padding-left:2%; 
 }
 
-#logo{
+#logo {
+  z-index: 4;
+  font-family: 'Poiret One', sans-serif;
+  display:flex; 
+  word-wrap: break-word;
+  align-items: flex-end;  
+  size:inherit;
   float: left;
-  font-size: 6.5vh;
-  display: block;
+  bottom:2vh; 
+  margin:0;
+  position: fixed;
+  font-size: 5vw;
   color: #f2f2f2;
-  text-align: center;
-  padding: 5px 10px;
+  text-align: left;
   text-decoration: none;
 }
-
-
+#title {
+  margin: 18vh 0vw 3vh 0vw;
+  text-align: center;
+  font-size: 10vh;
+  color: rgb(37, 51, 51); 
+}
+h1 {
+    font-family: 'Open Sans', sans-serif;
+}
 .textBox {
-  padding: 5vh 3vw 20vh 3vw;
-  border-radius:15px 50px; 
+  margin-top: 0vh;
+  margin-bottom: 15vh;
   font-family: 'Open Sans', sans-serif;
   text-align: center; 
-  font-size: 4vh;
+  font-size: 2.5vh;
   color: rgb(37, 51, 51);
-  width: 90vw;
-  height: 20vh; 
 }
+/*ending of style for top part of page*/
 
-.bio {
+/*Beginning of style for the expandable grid*/
+ .row {
+  border-radius: 5vw;
+  position: relative;
+  width: 64vw;
+  display: flex;
+  flex-direction: row;
+  overflow: hidden;
+  justify-content: center;
+  background: none;
+  margin: 0 auto;
+ }
+.pictureBox {
```

Comment:
I was so interested on why there are so many properties:
After playing with the code for a while here are my conclusions:

The logic in here is so brilliant, the way you created the grid or the triangle, so nice,
The thing is that CSS now have properties for things like these, so we do not longer need this hacks:
Here is the result of me playing for a while:
```
/*Beginning of style for the expandable grid*/
 .row {
  width: 64vw;
  margin: 1rem auto;
  display: grid;
  grid-template-columns: 1fr 1fr;
  grid-auto-flow: column;  
  height: 40vw;
  grid-gap: 1rem;
 }

.pictureBox {
  border-radius: 2vw;
  background-size: cover; 
  cursor: pointer;
}

#triangle { }

#name{  
  color: rgb(229, 231, 225);
  z-index: 2; 
  font-weight: bold;
  font-family: 'Poiret One', sans-serif;
  font-size: 6vh;
  vertical-align: top;
  border-top-left-radius: 2vw;
  width: 26vh;
  height: 14vh;
  padding: 0.7rem;
  background: linear-gradient(to right bottom, #3553129a 50%, #ffffff00 50%);
}
```

Note that we are using css grid (really important topic)
and linear-gradient.
Delete a lot of properties that was set to their default

Here are some pics:
<img width="1440" alt="Screen Shot 2020-04-12 at 2 57 48 PM" src="https://user-images.githubusercontent.com/23039467/79078540-a1548700-7cce-11ea-91a9-720ee0d45c3e.png">


<img width="1440" alt="Screen Shot 2020-04-12 at 2 58 04 PM" src="https://user-images.githubusercontent.com/23039467/79078545-a6193b00-7cce-11ea-9294-d8d63648f7ac.png">

They give more or less the same result with a lot less but clearer code

Reply:
This is not perfect, I just played with the code for 20 minutes, but if you are interested in this ping me